### ISSUE TITLE
get JSONLD context with conneg doesn't work

### ISSUE THREAD

**Issue by VladimirAlexiev** [schemaorg/schemaorg#2851]
```
curl -L -Haccept:application/ld+json https://schema.org/
Warning: Binary output can mess up your terminal. Use "--output -" to tell
Warning: curl to output it to your terminal anyway, or consider "--output
Warning: <FILE>" to save to a file.
```

Save that binary file:
```
curl -L -Haccept:application/ld+json https://schema.org/ > file.gz
```

Uncompress it and you'll find it's a gzipped version of the Schema home page.
(has `<h1>Welcome to Schema.org</h1>`)

Let's check the headers:
```
curl -I -L -Haccept:application/ld+json https://schema.org/
HTTP/2 200
access-control-allow-credentials: true
access-control-allow-headers: Accept
access-control-allow-methods: GET
access-control-allow-origin: *
access-control-expose-headers: Link
link: </docs/jsonldcontext.jsonld>; rel="alternate"; type="application/ld+json"
date: Fri, 26 Feb 2021 11:07:52 GMT
expires: Fri, 26 Feb 2021 11:17:52 GMT
etag: "v7F5aA"
x-cloud-trace-context: b911bb393a7606b0fd59507bb253b849
content-type: text/html
content-encoding: gzip
server: Google Frontend
content-length: 2235
age: 455
cache-control: public, max-age=600
```

`</docs/jsonldcontext.jsonld>` is the appropriate JSONLD context.

I don't know enough about links to tell whether this should force the browser to fetch the proper file:
```
link: </docs/jsonldcontext.jsonld>; rel="alternate"; type="application/ld+json"
```

But it seems to me the response should be a 303 with `Location` header, not 200 with this `link` header.

**Comment by RichardWallis** [comment]
Content negotiation is not supported on the Schema.org site.

See comment for detailed description as to why: https://github.com/schemaorg/schemaorg/issues/2578#issuecomment-632227864

As per the JSON-LD 1.1 standard machine readable access is indicated via the link value:
```link: </docs/jsonldcontext.jsonld>; rel="alternate"; type="application/ld+json"```

**Comment by VladimirAlexiev** [comment]
Close this as duplicate of #2578

**Comment by michielhildebrand** [comment]
We believe this is not a duplicate of #2578 as this issues (also) relates to the content-encoding header, which is not discussed in #2578. 

The server shouldn't send content-encoding: gzip without the client having signaled that it is acceptable, as is described in a related issue on the curl repo https://github.com/curl/curl/issues/2836. 

BTW. We noticed that the `content-encoding: gzip` is not always included in the response. It varied per ip-address that we requested from.
